app/controllers/v0/disability_compensation_forms_controller.rb

```diff
@@ -105,6 +104,10 @@ def auth_rating_info
       authorize(api, :rating_info_access?)
     end
 
+    def form_content
+      @form_content ||= JSON.parse(request.body.string)
```

Comment:
The form content already exists on the SavedClaim object (`saved_claim.form_data`) with a convenience method `saved_claim.parsed_form` 
I'd prefer to reuse existing data unless there is a reason not to.

Reply:
@tblackwe this method is being used before the `SavedClaim` is created. 
this method is used to save the data on line 57!
and then the memoized `@form_content` is used in the `create_submission` -> `includes_toxic_exposure?` methods later.

Reply:
I agree it is currently being called before savedclaim is created, but the data is not used until later, and in the `create_submission` method where we have already passed in the saved claim object

Reply:
re-reading Aurora's comment and getting myself confused.

If ya'll think the code is fine as is, I dont feel strongly. But I do think we are repeating ourselves by parsing the form from 2 different places.

Reply:
I think there's a differentiation between a saved claim's form data being parsed and the request body's content, which is necessary to crated the saved claim. Memoizing, the data is parsed once.

The suggested refactor is viable, although slightly less readable and would require a second parsing. Also, the memoized instance variable, and its cached result, is already available.

Reply:
I'm good with the explanation, and happy for you to leave it as you have it. Thanks for taking a second look